docs/capabilities.md

```diff
@@ -0,0 +1,70 @@
+Sematic is designed to support arbitrarily complex pipelines of Python-defined
+business logic running on heterogeneous compute. Pipeline steps can notably include:
+
+* **Data processing** – Apache Spark jobs, Google Dataflow jobs, or other map/reduce jobs
```

Comment:
Do we want to call out Google Dataflow before having explicit integration support?